lib/browser/api/menu-item-roles.js

```diff
@@ -254,7 +254,8 @@ exports.getDefaultAccelerator = (role) => {
 }
 
 exports.shouldRegisterAccelerator = (role) => {
-  return roles.hasOwnProperty(role) ? roles[role].registerAccelerator : true
+  const hasRoleRegister = roles.hasOwnProperty(role) && roles[role].registerAccelerator
```

Comment:
This will be `false` if `roles[role].registerAccelerator` is not defined (correct), but also if it's been explicitly set to `false`.

That's exactly how this is actually used unfortunately - to disable registering accelerators on the cut/copy/paste/etc roles - so this will cause some problems.

Needs to be something like `roles[role].hasOwnProperty('registerAccelerator')` instead I think.

Reply:
oh oops i meant to check explicitly that it's `undefined` - fixing